modules/ROOT/pages/questions/question-21.adoc

```diff
@@ -0,0 +1,23 @@
+21. `mceInsertTable`` -> `mceInsertTableDialog`
```

Comment:
@lnewson

Reply:
This wasn't related to my change, so tagging the right person: @ltrouton

Reply:
It's worth noting that the old `mceInsertTable` command still exists in v6. In v5, the command was serving two purposes: opening the insert/edit table dialog or inserting a table without using a dialog if additional arguments were provided. For example (old way):
```
tinymce.activeEditor.execCommand('mceInsertTable'); // Opens dialog
tinymce.activeEditor.execCommand('mceInsertTable', false, { rows: 2, columns: 2 }); // Inserts table into content directly without opening a dialog
```
It was not ideal that the `mceInsertTable` command was serving two purposes so the `mceInsertTableDialog` command was created and the opening dialog functionality was removed from `mceInsertTable` command. For example (new way):
```
tinymce.activeEditor.execCommand('mceInsertTableDialog'); // Opens dialog
tinymce.activeEditor.execCommand('mceInsertTable', false, { rows: 2, columns: 2 }); // Inserts table into content directly
```
So basically in the migration guide, we will need to mention that:
> The `mceInsertTable` command can no longer open the insert table dialog. Use the `mceInsertTableDialog` command instead

so integrators know to use the new command if they were previously using the `mceInsertTable` command to open the table dialog